src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.logo_link {
+  height: 40px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__menu {
+  display: flex;
+  list-style: none;
+  height: 60px;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
You've styled the `.nav__menu` class using the tag name in your CSS. This is generally not recommended, as it can lead to issues with specificity and reusability. It's better to use class names for styling.